constellations/gemini-admin/src/emailPayment/accountInfo.js

```diff
@@ -0,0 +1,151 @@
+"use strict";
+require("../account/accountInfo.sass");
+import React from "react";
+import { Link } from "react-router-dom";
+import { getFullName } from "../utils/userUtils.js";
+
+export class AccountInfo extends React.Component {
+    state = {
+        selectedUsers: [],
+        selectedUserEmails: [],
+    };
+
+    formatCurrency = (amount) => {
+        return new Intl.NumberFormat("en-US", {
+            style: "currency",
+            currency: "USD",
+        }).format(amount);
+    };
+
+    onCheckUser = (e, userId) => {
+        const users = this.props.users;
+        if (e.target.checked) {
+            var emails = this.state.selectedUserEmails;
+            var currentSelectedUsers = this.state.selectedUsers;
+            const checkedUser = users.find((user) => user.id == userId);
+            emails.push(checkedUser.email);
+            currentSelectedUsers.push(checkedUser);
+            this.setState({
+                selectedUserEmails: emails,
```

Comment:
I think let's get rid of this. We already keep track of the selected Users.
If you ever want only emails for selected Users, you can always use `selectedUsers.map(user => user.email)` which will create a list of emails from the list of users.

Having two variables of similar things and always having to update both can be messy and introduces potential problems like forgetting to update one of them.